ATTRIBUTIONS/wasabi-email-attribution.xml

```diff
@@ -1214,7 +1214,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/jwu/git/wasabi-core/modules/email/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```

Comment:
@shoeffner good catch. @jwtodd, any idea how this string was generated from attribution maven plugin?